views/js/pciManager/tpl/listing.tpl

```diff
@@ -7,6 +7,8 @@
 
     <div class="actions">
         <div class="switch-box"></div>
+        <div class="unregister"></div>
```

Comment:
I wouldn't bother including these 2 extra `<div>`s after the switch box. It's just extra html. Instead we can render the buttons directly into the outer `actions` div.